cypress/integration/visitorCanSeeMainPage.feature.js

```diff
@@ -0,0 +1,37 @@
+describe('Visitor can see main page', () => {
+  describe('success', () => {
+    beforeEach(() => {
+      cy.intercept('GET', 'https://fake-newzzzz.herokuapp.com/api/articles', {
+        fixture: 'articles.json',
+      });
+      cy.visit('/');
+    });
+    it('displays main page layout', () => {
+      cy.get('[data-cy=navbar]').should('exist');
+      cy.get('[data-cy=header]').should('exist');
+      cy.get('[data-cy=breaking-news]').should('exist');
+      cy.get('[data-cy=breaking-img]').should('exist');
+      cy.get('[data-cy=breaking-img]').should('exist');
+      cy.get('[data-cy=breaking-title]').should('contain', 'Something');
+      cy.get('[data-cy=breaking-teaser]').should('contain', 'Something more');
+      cy.get('[data-cy=breaking-date]').should('contain', '2021-05-12');
+      cy.get('[data-cy=articles-container]').should('exist');
+      cy.get('[data-cy=footer]').should('exist');
+    });
+  });
+  describe('unsuccessfully', () => {
+    before(() => {
+      cy.intercept('GET', 'https://fake-newzzzz.herokuapp.com/api/articles', {
+        statusCode: 500,
+        error: '500 Internal Server Error |  0     bytes\n',
```

Comment:
Why `  |  0     bytes\n` ?

Reply:
We had this same way since slow food, wasn't sure why, now it's gone :)